source/wp-content/themes/wporg-parent-2021/patterns/two-columns-with-cta.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Title: 2 Columns with CTA
+ * Slug: wporg-parent-2021/two-columns-with-cta
+ * Categories: columns, buttons
+ */
+
+?>
+<!-- wp:columns {"align":"full","style":{"border":{"top":{"color":"var:preset|color|light-grey"}}},"className":"is-style-default"} -->
+<div class="wp-block-columns alignfull is-style-default" style="border-top-color:var(--wp--preset--color--light-grey)"><!-- wp:column {"width":"100%","style":{"spacing":{"padding":{"top":"80px","right":"100px","bottom":"100px","left":"100px"}}}} -->
+<div class="wp-block-column" style="padding-top:80px;padding-right:100px;padding-bottom:100px;padding-left:100px;flex-basis:100%"><!-- wp:heading {"level":4} -->
+<h4>Download and install yourself</h4>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>For anyone comfortable getting their own hosting and domain.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button {"style":{"spacing":{"padding":{"right":"32px","left":"32px"}}}} -->
+<div class="wp-block-button"><a class="wp-block-button__link wp-element-button" style="padding-right:32px;padding-left:32px">Download WordPress 6.0</a></div>
+<!-- /wp:button -->
+
+<!-- wp:button {"textColor":"blue-1","style":{"spacing":{"padding":{"left":"32px","right":"32px"}}},"className":"is-style-outline"} -->
+<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-blue-1-color has-text-color wp-element-button" style="padding-right:32px;padding-left:32px">Installation guide</a></div>
+<!-- /wp:button --></div>
+<!-- /wp:buttons -->
+
+<!-- wp:paragraph {"textColor":"dark-strokes-grey","fontSize":"tiny"} -->
+<p class="has-dark-strokes-grey-color has-text-color has-tiny-font-size">Recommend PHP 7.4 or greater and MySQL 5.7 or MariaDB version 10.3 or greater.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"100%","style":{"spacing":{"padding":{"top":"80px","right":"100px","bottom":"80px","left":"100px"}},"border":{"left":{"color":"var:preset|color|light-grey"}}}} -->
+<div class="wp-block-column" style="border-left-color:var(--wp--preset--color--light-grey);padding-top:80px;padding-right:100px;padding-bottom:80px;padding-left:100px;flex-basis:100%"><!-- wp:heading {"level":4} -->
```

Comment:
This border style will need to be switched from left to top on small screens

Reply:
Also this border isn't truly centered on the screen due to the column gap

![Screen Shot 2022-07-29 at 8 52 04 AM](https://user-images.githubusercontent.com/1017872/181635425-90695ddb-5dc3-458f-974f-d75c62cd00d0.jpg)

We'd need CSS to correct this, or adjust via theme/block settings maybe?

Reply:
If we wanted to keep this within the editor, you can create a column in between the two and set its width to `1px` with the background color you want. With the current Gutenberg style, it would disappear on mobile and probably be close enough to the design. For me, the visual divider on mobile would be distracting.

Reply:
I don't know if this matters, but I get block errors when inserting this into a post:
<img width="1823" alt="Screen Shot 2022-07-29 at 3 12 19 pm" src="https://user-images.githubusercontent.com/7200686/181687332-79a8c3d1-f405-4c44-b8b8-77e2d1aa238a.png">

Reply:
@tellyworth I can't repro this but I have seen this error before during my support rotation and I'm sure there was an existing issue for it... I'll try to find it

Reply:
I did not run into that issue when I was testing.